annotate.el

```diff
@@ -627,7 +627,9 @@ an overlay and it's annotation."
 
 (defun annotate-load-annotation-data ()
   "Read and return saved annotations."
+  (setq local-annotate-file annotate-file)
```

Comment:
I added a comment and probably should read a bit more about Lisp variables. All changes are in 135fe586743ad4759d4bcfe53bbb343953c2e841.